src/YesSql.Provider.MySql/MySqlDbProviderOptionsExtensions.cs

```diff
@@ -27,7 +35,7 @@ public static class MySqlDbProviderOptionsExtensions
             {
                 ConnectionFactory = new DbConnectionFactory<MySqlConnection>(connectionString, true),
```

Comment:
This should use the default value for `reuseConnection`. Meaning you can remove the `true`. Do this on all extension methods.